src/test/kotlin/app/batch/S3DirectoryWriterTest.kt

```diff
@@ -0,0 +1,67 @@
+package app.batch
+
+import com.amazonaws.services.s3.AmazonS3Client
+import com.amazonaws.services.s3.model.PutObjectRequest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.context.junit4.SpringRunner
+
+@RunWith(SpringRunner::class)
+@ActiveProfiles("phoneyDataKeyService", "phoneyCipherService", "unitTest", "outputToS3")
+@SpringBootTest
+@TestPropertySource(properties = [
+    "directory.output=ephemera",
+    "output.batch.size.max.bytes=100000",
+    "source.table.name=ucdata",
+    "compress.output=true",
+    "encrypt.output=true",
+    "aws.region=eu-west-1",
+    "s3.bucket=not_set",
+    "s3.prefix.folder=not_set"
+])
+class S3DirectoryWriterTest {
```

Comment:
Blank line please

Reply:
Hi Mark ,  i havent run with integration-all on this branch , will do . Also on the formatting , Do we have any standard formatter used in thhe project which i can  import in intellij editor

Reply:
Mark , I have now fixed the suggested formatting and also ran make destroy integration-all which is successful as well